src/powershell/Public/Get-FinOpsCostExport.ps1

```diff
@@ -0,0 +1,150 @@
+
+    .PARAMETER Name
+        Name of the Cost Management Export.
+
+    .PARAMETER Scope
+        The scope associated with export operations. 
+        This includes '/subscriptions/{subscriptionId}/' for subscription scope, 
+        '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, 
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope and 
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, 
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope, 
+        '/providers/Microsoft.Management/managementGroups/{managementGroupId} for Management Group scope,
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billingProfile scope, 
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/invoiceSections/{invoiceSectionId}' for invoiceSection scope, and 
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/customers/{customerId}' specific for partners.
+
+    .PARAMETER DataSet
+        Optional. Dataset to get exports for. Allowed values = "ActualCost", "AmortizedCost", "Usage". Default = null (all exports).
+
+    .PARAMETER StorageAccountId
+        Optional. Resource ID of the storage account to get exports for. Default = null (all exports).
+
+    .PARAMETER StorageContainer
+        Optional. Name of the container to get exports for. Default = null (all exports).
+
+    .PARAMETER ApiVersion
+        Optional. API version to use when calling the Cost Management Exports API. Default = 2023-03-01.
+
+    .EXAMPLE
+
+        Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000"
+        Get-FinOpsCostExport -Name MyExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000"
+        Get-FinOpsCostExport -Name MyExport* -Scope "/subscriptions/00000000-0000-0000-0000-000000000000"
+        Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -DataSet "AmortizedCost"
+        Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageAccountId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Storage/storageAccounts/MyStorageAccount"
+        Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageContainer "MyContainer"
+        Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageContainer "mtd*" -apiVersion "2023-08-01"
+        
+#>
+function Get-FinOpsCostExport
+{
+    param
+    (
+        [Parameter()]
+        [SupportsWildcards()][string]
+        $Name = $null,
+
+        [Parameter(Mandatory = $true)]
```

Comment:
Can you make this optional and default to the subscription scope from AzContext?

Reply:
done